funding/bin/utils.py

```diff
@@ -72,7 +72,8 @@ def price_cmc_btc_usd():
 def coin_btc_value():
     headers = {'User-Agent': 'Mozilla/5.0 (Android 4.4; Mobile; rv:41.0) Gecko/41.0 Firefox/41.0'}
     try:
-        r = requests.get('https://bittrex.com/api/v1.1/public/getmarketsummary?market=btc-aeon', headers=headers)
+        r = requests.get('https://bittrex.com/api/v1.1/public/getmarketsummary?market=btc-' +
+                         settings.COINTICKER, headers=headers)
```

Comment:
COINTICKER -> COINCODE? 

Perhaps renaming COINCODE in settings to COINTICKER would be more appropriate. Thoughts?